README.md

```diff
@@ -4,45 +4,54 @@ pip-audit
+`pip-audit` is a prototype tool for scanning Python environments for packages
+with known vulnerabilities. It uses the Python Packaging Advisory Database
+(https://github.com/pypa/advisory-db) as a source of vulnerability reports.
 
-This project is developed by [Trail of Bits](https://www.trailofbits.com/) with support from Google. This is not an official Google product.
+This project is developed by [Trail of Bits](https://www.trailofbits.com/) with
+support from Google. This is not an official Google product.
 
-## Development steps
+## Installation
 
-```bash
-git clone https://github.com/trailofbits/pip-audit && cd pip-audit
-make dev && source env/bin/activate
-pip-audit --help
-```
-
-## Release process
-
-Releases of `pip-audit` are managed via [`bump`](https://github.com/di/bump)
-and GitHub Actions.
+`pip-audit` requires Python 3.6 or newer, and can be installed directly via
+`pip`:
 
 ```bash
-# default release (patch bump)
-make release
-
-# override the default
-# vX.Y.Z -> vX.Y.Z-rc.0
-make release BUMP_ARGS="--pre rc.0"
-
-# vX.Y.Z -> vN.0.0
-make release BUMP_ARGS="--major"
+pip install pip-audit
 ```
 
-`make release` will fail if there are any untracked changes in the source tree.
-
-If `make release` succeeds, you'll see an output like this:
+## Usage
 
 ```
-RUN ME MANUALLY: git push origin main && git push origin vX.Y.Z
+usage: pip-audit [-h] [-V] [-l] [-r REQUIREMENTS] [-f {columns,json}]
+                 [-s {osv,pypi}] [-d] [--desc {on,off,auto}]
+                 [--cache-dir CACHE_DIR]
+
+audit the Python environment for dependencies with known vulnerabilities
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -V, --version         show program's version number and exit
+  -l, --local           show only results for dependencies in the local
+                        environment (default: False)
+  -r REQUIREMENTS, --requirement REQUIREMENTS
+                        audit the given requirements file; this option can be
+                        used multiple times (default: None)
+  -f {columns,json}, --format {columns,json}
+                        the format to emit audit results in (default: columns)
+  -s {osv,pypi}, --vulnerability-service {osv,pypi}
+                        the vulnerability service to audit dependencies
+                        against (default: pypi)
+  -d, --dry-run         collect all dependencies but do not perform the
+                        auditing step (default: False)
+  --desc {on,off,auto}  include a description for each vulnerability; `auto`
+                        only includes a description for the `json` format
+                        (default: auto)
+  --cache-dir CACHE_DIR
+                        the directory to use as an HTTP cache for PyPI; uses
+                        the `pip` HTTP cache by default (default: None)
 ```
 
-Run that last command sequence to complete the release.
-
```

Comment:
Let's also add a "Contributing" section here and link to the new contributing docs.

Reply:
101a306